web-demo/doodle_game/backend/src/main/java/ai/djl/examples/doodle/Handler.java

```diff
@@ -63,14 +61,18 @@ public void handleRequest(InputStream is, OutputStream os, Context context) thro
                 ImageFactory factory = ImageFactory.getInstance();
                 img = factory.fromInputStream(bis);
             }
-            System.setProperty(
-                    "ai.djl.repository.zoo.location",
-                    "https://alpha-djl-demos.s3.amazonaws.com/model/quickdraw/doodle_mobilenet.zip");
+
+            Translator<Image, Classifications> translator =
+                    ImageClassificationTranslator.builder()
```

Comment:
Do we apply softmax by default in the end?

Reply:
No, I added softmax back.